REMARKs/AiyagariIdiosyncratic/README.md

```diff
@@ -0,0 +1,68 @@
+1. Please open this notebook in *JupyterLab*. To install it with pip, please conduct:
+
+   * pip install jupyterlab
+
+2. This notebook uses *Dolo* and *Dolark* to replicate Aiyagari(1994). To install Dolark, please follow the steps below: 
+
+   * git clone "dolo", "dolang" and "dolark" to the local directory:
+
+     ​    git clone https://github.com/econforge/dolo.git  
+     ​    git clone https://github.com/econforge/dolang.git   
+     ​    git clone https://github.com/econforge/dolark.git
+
+   *  go to the local directory and do "pip install -e . " for each of the three. (Three times in total) 
+
+     ​    pip install -e .   
+
```

Comment:
This can be replaced with
```
$ pip install git+https://github.com/econforge/dolo
$ pip install git+https://github.com/econforge/dolang
$ pip install git+https://github.com/econforge/dolark
```

instead of cloning the libraries locally.